CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ## [10.1.0] IN PROGRESS
 
+* Make Inventory search and browse query boxes expandable. Refs UIIN-2493.
```

Comment:
Should the `Browse` query box be expandable?

Reply:
Fixed in https://github.com/folio-org/stripes-acq-components/pull/722